__tests__/tests.test.js

```diff
@@ -159,3 +159,50 @@ describe("/api/articles/", () => {
       });
   });
 });
+
+//task 6
+describe("api/articles/:article_id/comments", () => {
+  test("get 200, and return sorted array of comments of required article_id by created_at in descending order by default", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body).toBe("object");
+        expect(res.body.comments.length).toBe(2);
+        expect(res.body.comments).toBeSortedBy("created_at", { descending: true });
+        res.body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at")
+        });
+      });
+  })
+  test("get 200, and return empty array if no comment for a valid request of article_id", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body).toBe("object");
+        
+      });
```

Comment:
This test description is really good, but you are not actually checking that the array is empty here.

Reply:
updated to check for []